scripts/scil_generate_sampling_scheme.py

```diff
@@ -0,0 +1,213 @@
+    p.add_argument('--duty',
+                   action='store_true',
+                   help='Apply duty cycle optimization. '
+                        '\nB-vectors are shuffled to reduce consecutive '
+                        'colinearity in the samples. [%(default)s]')
+    p.add_argument('--b0inter',
+                   dest='b0_every', type=int, default=-1,
+                   help='Interleave a b0 every b0_every. \nNo b0 if 0. '
+                        '\nOnly 1 b0 at beginning if > number of samples '
+                        'or negative. [%(default)s]')
+    p.add_argument('--b0end',
+                   action='store_true', dest='b0_end',
+                   help='Add a b0 as last sample. [%(default)s]')
+    p.add_argument('--b0value',
+                   dest='b0_value', type=float, default=0.01,
+                   help='b-value of the b0s. [%(default)s]')
+
+    bvals_group = p.add_mutually_exclusive_group(required=True)
+    bvals_group.add_argument('--bvals',
+                             type=float, nargs='+', metavar='bvals',
+                             help='bval of each non-b0 shell.')
+    bvals_group.add_argument('--blinmax',
+                             dest='b_lin_max', type=float,
+                             help='b-max for linear bval distribution '
+                                  'in *b*. [replaces -bvals]')
+    bvals_group.add_argument('--qlinmax',
+                             dest='q_lin_max', type=float,
+                             help='b-max for linear bval distribution '
+                                  'in *q*. [replaces -bvals]')
+
+    g1 = p.add_argument_group(title='Save as')
+    g1.add_argument('--caru',
+                    action='store_true',
+                    help='Save in caruyer format (.caru). [%(default)s]')
+    g1.add_argument('--phil',
+                    action='store_true',
+                    help='Save in Philips format (.txt). [%(default)s]')
+    g1.add_argument('--fsl',
+                    action='store_true',
+                    help='Save in FSL format (.bvecs/.bvals). [%(default)s]')
+    g1.add_argument('--siemens',
+                    action='store_true',
+                    help='Save in Siemens format (.dvs). [%(default)s]')
+    g1.add_argument('--mrtrix',
+                    action='store_true',
+                    help='Save in MRtrix format (.b). [%(default)s]')
+
+    add_verbose_arg(p)
+    add_overwrite_arg(p)
+
+    return p
+
+
+def main():
+
+    parser = _build_args_parser()
+    args = parser.parse_args()
+
+    caru = args.caru
+    phil = args.phil
+    fsl = args.fsl
+    siemens = args.siemens
+    mrtrix = args.mrtrix
+
+    if not (caru or phil or fsl or siemens or mrtrix):
+        parser.error('Select at least one save format.')
+        return
+
+    if args.verbose:
+        logging.basicConfig(level=logging.DEBUG)
+    else:
+        logging.basicConfig(level=logging.INFO)
```

Comment:
I don't like the fact that you always have text displayed even if the verbose version is not activated.